sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -499,8 +500,48 @@ case class TruncateTableCommand(
         val path = new Path(location.get)
         try {
           val fs = path.getFileSystem(hadoopConf)
+          val fileStatus = fs.getFileStatus(path)
+          // Not all fs impl. support these APIs.
+          var optPermission: Option[FsPermission] = None
+          try {
+            optPermission = Some(fileStatus.getPermission())
+          } catch {
+            case NonFatal(e) =>
```

Comment:
@cloud-fan @HyukjinKwon By using try catch, I'm not sure what we should do for exception during getPermission/getAcls. I think we don't/can't do anything?

Reply:
In this case, I guess `case NonFatal(_) => // do nothing` is okay.